src/main/java/control/CreationModalViewController.java

```diff
@@ -121,13 +124,13 @@ private void createCameraShot(MouseEvent event) {
             TimelineController timelineController = this.controllerManager.getTimelineControl();
 
             cameraShotCreationModalView.getCamerasInShot().forEach(cameraIndex -> {
-                    timelineController.addCameraShot(cameraIndex,
+                    timelineController.addCameraShot(cameraIndex, new CameraShot(
```

Comment:
for readability it might be better to define this camerashot on a seperate line before it is passed to addCameraShot